codedeps.yaml

```
@@ -1434,3 +1434,42 @@ data:
         //CMAKE_INSIST_FIND_PACKAGE_Einsums: true
         //CMAKE_DISABLE_FIND_PACKAGE_Einsums: true
 
+  - project: gauxc 
+    use:
+      added: 1.10
+      required: false
+      required_note: "Allow using the GauXC density functional library."
+      buildable: true
+      buildtime: true
+      cms: true
+    repository:
+      host: github
+      account: wavefunction91
+      name: GauXC
+      commit: 6a8f4bf
+    #cmake: 
+    #  name: gauxc
+    #  constraint: null
+    #  target: gauxc::gauxc
+    #  components:
+    #  ENABLE_gauxc
+    #  gauxc_ENABLE_GPU 
+    conda: null
+
+  - project: h5py 
+    use:
+      added: 1.10
+      required: false
+      required_note: optional for GauXC interface tests
+      buildable: false
+      buildtime: false
+      cms: false
+      test_required: false
+    repository: null
+    cmake: null
+    conda:
+      channel: conda-forge
+      name: h5py
+      constraint: null
+      cmake: null
```

```suggestion
      cmake: null
      cmake_note: "Primarily OTF runtime detected."
```
this phrase used by path advisor script for grouping